util/src/main/java/io/kubernetes/client/leaderelection/resourcelock/ResourceLockConfig.java

```diff
@@ -0,0 +1,40 @@
+package io.kubernetes.client.leaderelection.resourcelock;
+
+/**
+ * ResourceLockConfig common data that exists across different
+ * resource locks
+ */
+public class ResourceLockConfig {
+    /**
+     * Identity is the unique string identifying a lease holder across
+     * all participants in an election.
+     */
+    private String identity;
+    /**
+     * EventRecorder is optional.
+     */
+    private EventRecorder eventRecorder;
```

Comment:
remove this. we should design/implement event utilities in another pull. i updated the issue to reflect the task.

Reply:
`EventRecorder`is just a interface for ResourceLock to record the event, 
[`client-go/leaderelection/resroucelock/interface.go`](https://github.com/kubernetes/client-go/blob/master/tools/leaderelection/resourcelock/interface.go#L53),

Reply:
don't design it if it doesn't intend to use it

Reply:
OK, i will delete this interface.